app/utils/UserAgentFilter.scala

```diff
@@ -10,25 +10,24 @@ import scala.concurrent.Future
 
 trait UserAgentFilter {
 
-  protected val configuration : ServiceConfiguration
+  protected val configuration: ServiceConfiguration
 
   private val userAgents: Seq[String] = configuration.allowedUserAgents
 
-  def onlyAllowedServices[A](block: (Request[A], String) => Future[Result])
-                            (implicit request: Request[A]): Future[Result] = {
-
+  def onlyAllowedServices[A](block: (Request[A], String) => Future[Result])(
+    implicit request: Request[A]): Future[Result] =
     request.headers.get(HeaderNames.USER_AGENT) match {
       case Some(userAgent) if allowedUserAgent(userAgent) =>
         block(request, userAgent)
-      case _ => {
-        Logger.warn(s"Invalid User-Agent: [${request.headers.get(HeaderNames.USER_AGENT)}].")
+      case userAgent =>
```

Comment:
minor point (feel free to ignore). I think userAgent is an an option here. Does it make sense to fold over the option in the warning to avoid:

`Logger.warn(s"Invalid User-Agent: Some(xxx).")`

Reply:
that's a good point and I did notice this, but then original code did `${request.headers.get(HeaderNames.USER_AGENT)}` which would output Some(xxx) and my intentions in this PR were to not change how the application behaves